CONTRIBUTING.md

```diff
@@ -16,15 +16,15 @@ If possible, please also add Acceptance Criteria so it is clear what is required
 
 ## Making Changes
 
-The production website, https://metabolicatlas.org, is based on the `main` branch. For development, new work should be based on the `develop` branch.
+The production website, https://metabolicatlas.org, is based on the `main` branch. All new development should be based on this branch.
 
 ### Getting Started
 
 To get started with running the project on your local development machine, please follow the [README](https://github.com/MetabolicAtlas/MetabolicAtlas/blob/develop/README.md)
```

Comment:
```suggestion
To get started with running the project on your local development machine, please follow the [README](https://github.com/MetabolicAtlas/MetabolicAtlas/blob/main/README.md)
```

Reply:
Similarly, the link in the `README` should also be changed. The last sentence in the screenshot could probably be removed.

<img width="735" alt="image" src="https://user-images.githubusercontent.com/423498/173053079-0c8584be-9026-4efc-8c7a-fd8e04876838.png">

Reply:
Nice catch, sloppy that I did not find it myself. Fixed in 4271d4256c54d1b3b0ca3b818322ecbd350cb399